packages/python-sdk/scripts/generate_api_ref.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This script generates the python sdk api reference markdown files
+# Run it in the `python-sdk/` directory
+
+packages=("sandbox_sync" "sandbox_async" "exceptions")
+
+mkdir api_ref
+
+for package in "${packages[@]}"; do
+    # generate raw api reference markdown file
+    pydoc-markdown -p e2b.${package} >api_ref/${package}.mdx
+    # remove package path display
+    sed -i '' '/<a[^>]*>.*<\/a>/d' api_ref/${package}.mdx
+    # remove empty hyperlinks
+    sed -i '' '/^# /d' api_ref/${package}.mdx
+    # move to docs (for later use)
```

Comment:
is there anything wrong with it right now? Also feel free to add the generated files. Could you also add the route to `.gitattributes` 🙏🏻 (feel free to ignore this if the route isn't decided yet)

Reply:
I remove the package paths bc `pydoc-markdown` includes all paths, even for packages that are not documented

Reply:
and the naming becomes redundant with this as well IMO